PanelTranslator@klangman/files/PanelTranslator@klangman/languages_0_9_6_12.js

```diff
@@ -0,0 +1,130 @@
+   {code:"ko",      englishName:"Korean"},
+   {code:"sw",      englishName:"Swahili"},
+   {code:"hr",      englishName:"Croatian"},
+   {code:"ku",      englishName:"Kurdish"},
+   {code:"sv",      englishName:"Swedish"},
+   {code:"cs",      englishName:"Czech"},
+   {code:"ky",      englishName:"Kyrgyz"},
+   {code:"ty",      englishName:"Tahitian"},
+   {code:"da",      englishName:"Danish"},
+   {code:"lo",      englishName:"Lao"},
+   {code:"tg",      englishName:"Tajik"},
+   {code:"nl",      englishName:"Dutch"},
+   {code:"la",      englishName:"Latin"},
+   {code:"ta",      englishName:"Tamil"},
+   {code:"mhr",     englishName:"Eastern Mari"},
+   {code:"lv",      englishName:"Latvian"},
+   {code:"tt",      englishName:"Tatar"},
+   {code:"emj",     englishName:"Emoji"},
+   {code:"lt",      englishName:"Lithuanian"},
+   {code:"te",      englishName:"Telugu"},
+   {code:"en",      englishName:"English"},
+   {code:"lb",      englishName:"Luxembourgish"},
+   {code:"th",      englishName:"Thai"},
+   {code:"eo",      englishName:"Esperanto"},
+   {code:"mk",      englishName:"Macedonian"},
+   {code:"to",      englishName:"Tongan"},
+   {code:"et",      englishName:"Estonian"},
+   {code:"mg",      englishName:"Malagasy"},
+   {code:"tr",      englishName:"Turkish"},
+   {code:"fj",      englishName:"Fijian"},
+   {code:"ms",      englishName:"Malay"},
+   {code:"udm",     englishName:"Udmurt"},
+   {code:"tl",      englishName:"Filipino"},
+   {code:"ml",      englishName:"Malayalam"},
+   {code:"uk",      englishName:"Ukrainian"},
+   {code:"fi",      englishName:"Finnish"},
+   {code:"mt",      englishName:"Maltese"},
+   {code:"ur",      englishName:"Urdu"},
+   {code:"fr",      englishName:"French"},
+   {code:"mi",      englishName:"Maori"},
+   {code:"uz",      englishName:"Uzbek"},
+   {code:"fy",      englishName:"Frisian"},
+   {code:"mr",      englishName:"Marathi"},
+   {code:"vi",      englishName:"Vietnamese"},
+   {code:"gl",      englishName:"Galician"},
+   {code:"mn",      englishName:"Mongolian"},
+   {code:"cy",      englishName:"Welsh"},
+   {code:"ka",      englishName:"Georgian"},
+   {code:"my",      englishName:"Myanmar"},
+   {code:"xh",      englishName:"Xhosa"},
+   {code:"de",      englishName:"German"},
+   {code:"ne",      englishName:"Nepali"},
+   {code:"yi",      englishName:"Yiddish"},
+   {code:"el",      englishName:"Greek"},
+   {code:"no",      englishName:"Norwegian"},
+   {code:"yo",      englishName:"Yoruba"},
+   {code:"gu",      englishName:"Gujarati"},
+   {code:"pap",     englishName:"Papiamento"},
+   {code:"yua",     englishName:"Yucatec Maya"},
+   {code:"ht",      englishName:"Haitian Creole"},
+   {code:"ps",      englishName:"Pashto"},
+   {code:"zu",      englishName:"Zulu"},
+   {code:"ha",      englishName:"Hausa"},
+   {code:"fa",      englishName:"Persian"},
+   {code:"haw",     englishName:"Hawaiian"},
+   {code:"pl",      englishName:"Polish"}
```

Comment:
You could change the `englishName` key to just `name` and then use translatable strings, e.g. `name: _('French')`. Translations will be very long but it would solve the issue.

Reply:
> If a newer translate-shell is install (newer then currently available in the default Mint repositories) then the English and non-English names are read directly from translate-shell and this hard-coded list is not used.

So this is essentially a short-term problem. The next version of Mint/Cinnamon will have 0.9.7 which is the same as LMDE. Could you branch a new version of this  in a 6.2 folder since everywhere will have that newer version of translate-shell at that time?

Reply:
Actually.. Do you happen to know of a GUI widget I can use that support St.Scrollable to act as a "text box" for the from/to test entries?  I tried to use St.ScrollView with an St.Entry as a child but it didn't work, just got scroll bars but no entry field.

Reply:
Maybe different params for the St.BoxLayout? Maybe @mtwebster has a better idea. I don't think I've tried to use the St class for something like that before.